tests/urql/atomWithSubscription.test.tsx

```diff
@@ -1,24 +1,21 @@
 import { Suspense } from 'react'
-import { render } from '@testing-library/react'
+import { fireEvent, render } from '@testing-library/react'
 import type { Client, TypedDocumentNode } from '@urql/core'
 import { interval, map, pipe, take, toPromise } from 'wonka'
-import { useAtom } from '../../src/'
+import { atom, useAtom } from '../../src/'
 import { atomWithSubscription } from '../../src/urql'
 import { getTestProvider } from '../testUtils'
 
-const withPromise = (source$: any) => {
```

Comment:
Removed as per `urql` docs:
https://formidable.com/open-source/urql/docs/advanced/subscriptions/#one-off-subscriptions

> This method will always returns a Wonka stream and doesn't have a .toPromise() shortcut method, since promises won't return the multiple values that a subscription may deliver.